examples/gnrc_border_router/README.md

```diff
@@ -217,7 +217,7 @@ USEMODULE += gnrc_slip
 ## Configuration
 
 In order to connect a RIOT 6LoWPAN border router over SLIP you run a small
-program called tunslip (imported from Contiki) [2] on the Linux host. The
+program called tunslip6 (imported from Contiki) [2] on the Linux host. The
```

Comment:
I don't get what you mean. The instructions for ethos are given at the beginning of this readme. For ethernet I don't know if we're giving explicit instructions or just saying that's a possibility.

Should we give _also_ instructions for an ethernet device?